src/sentry/static/sentry/app/views/releases/releaseCommits.jsx

```diff
@@ -83,7 +81,9 @@ const ReleaseCommits = React.createClass({
     return {
       loading: true,
       error: false,
-      commitList: []
+      commitList: [],
+      activeRepo: null,
+      title: 'All Repositories'
```

Comment:
ok, imo, i think it'd be better to not save the title separately in state and instead just in the render method do `this.state.activeRepo || 'All Repositories'` since they'd always be changed at the same time anyway